app/views/catalog/ds.html.erb

```diff
@@ -22,11 +22,8 @@
         # rights, display raw xml editing
     %>
       <%= if can?(:manage_item, @obj) &&
-             (
-               @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) ||
-               params[:dsid] == 'RELS-EXT'
-             ) &&
-             params[:dsid] != 'workflows'
+             @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) &&
```

Comment:
This line can go too.  All of the things in editable_datastreams meet this condition.

Reply:
Oh, that's great. Will simplify the test a bit as well. Thanks. Commit incoming.